spec/card_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'spec_helper'
+
+describe Marqeta::Card do
+  let(:pan) { '1234567890' }
+  let(:card_token) { 'CARD_TOKEN' }
+
+  describe 'class methods' do
+    describe '#from_pan' do
```

Comment:
Dot for class methods by convention: https://stackoverflow.com/questions/12764188/difference-between-and

Reply:
ah cool!

Reply:
@andyw8 fixed